dataactvalidator/filestreaming/csvAbstractReader.py

```diff
@@ -32,6 +32,7 @@ def open_file(self, region, bucket, filename, csv_schema, bucket_name, error_fil
         self.unprocessed = ''
         self.extra_line = False
         self.lines = []
+        self.flex_dictionary = {}
         self.header_dictionary = {}
```

Comment:
While we're here, does it make sense to replace these with lists? It looks like we're only indexing them by incrementing integer values.